syntax = "proto3";

package user;

service UserService {
    rpc GetUser(UserRequest) returns (UserReply);
}

message UserRequest {
    int32 id = 1;
}

message UserReply {
    int32 id = 1;
    string name = 2;
}
